Package.swift

```diff
@@ -0,0 +1,31 @@
+// swift-tools-version:5.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+import PackageDescription
+
+let package = Package(
+    name: "RxWebKit",
+    platforms: [
+        .iOS(.v8)
```

Comment:
Unless there is a specific need (such as APIs used), it's best to leave this _unspecified_ so that SwiftPM can set the minimum platform version to that which is supported by the toolchain.

This is especially important in the era of Xcode 12, as it drops support for iOS 8, making this package unusable with Xcode 12.